### ISSUE TITLE
Can we see multiple clients (from different processes) all at once in the profiler user interface?

### ISSUE THREAD

**Issue by asymingt** [wolfpld/tracy#520]
For robotics applications we often have multiple processes launched at once, which communicate over some messaging backbone, like ROS. It would be awesome to be able to profile all of these processes along a common timeline. Is this at all possible with tracy? My experience is that when I start two processes, there is a race condition and the first one connects to the server is the once that gets visualized, while the others are ignored.

I realize that I can use different ports, for example by setting the environment variable eg. `TRACY_PORT=8087` on each node. While this is great in the sense that I can spawn multiple tracy profiler servers (one to introspect each process) or switch a single UI between processes, it doesn't really give a holistic view of performance.

Could this be easily integrated into the current design, or could there maybe be a proxy that aggregates data across all clients to present itself as a single process to tracy server?

**Comment by wolfpld** [comment]
This is a duplicate of #471.

tl;dr Proxy is possible, but there are caveats.
